install/share/ipa-kdc-proxy.conf.template

```diff
@@ -23,8 +23,7 @@ WSGIScriptReloading Off
 
 <Location "/KdcProxy">
   Satisfy Any
-  Order Deny,Allow
-  Allow from all
+  Require all granted
```

Comment:
@rcritten Does the ``ipa-kdc-proxy.conf.template`` file require a ``# VERSION 2 - DO NOT REMOVE THIS LINE`` stanza now?

Reply:
Yes, it needs a version line so things will be upgraded. I suggested starting with 2 since version 1 was the un-versioned file.

Reply:
Hi @rcritten @tiran 
I have added the Version stanza in ipa-kdc-proxy.conf.template . Please review and let me know if this looks good and close this PR